.evergreen/config.yml

```diff
@@ -37,6 +37,17 @@ functions:
         working_dir: "src"
         script: |
            git submodule update --init
+    # Switch to a different version of libmongoc
+    - command: shell.exec
+      params:
+        working_dir: "src/src/libmongoc"
+        script: |
+           if [ "$LIBMONGOC_VERSION" != "" ]; then
```

Comment:
Is this more portable than `[ -z "$LIBMONGOC_VERSION" ]`?

Reply:
It's a habit of mine, but `[ -z ... ]` works just fine and achieves the same, so I'll change it.

Reply:
Correction, `-n` is what we're looking for.